internal/services/recoveryservices/site_recovery_protection_container_mapping_resource.go

```diff
@@ -92,11 +93,20 @@ func resourceSiteRecoveryProtectionContainerMapping() *pluginsdk.Resource {
 							Optional: true,
 							Default:  false,
 						},
+
 						"automation_account_id": {
 							Type:         pluginsdk.TypeString,
 							Optional:     true,
 							ValidateFunc: azure.ValidateResourceID,
-							RequiredWith: []string{"automatic_update.0.enabled"},
+						},
+
+						"authentication_type": {
+							Type:     pluginsdk.TypeString,
+							Optional: true,
+							Default:  string(replicationprotectioncontainermappings.AutomationAccountAuthenticationTypeSystemAssignedIdentity),
```

Comment:
what does that mean for existing resources? will this not change them as the default for them was different?

Reply:
Have updated code to make the default value effective in 4.0...